app/src/components/HomePage/HomePage.module.css

```diff
@@ -0,0 +1,28 @@
+.link {
+  display: flex;
+  margin: 5vh 0;
```

Comment:
For things like this, I would rather not use vh. Imagine if the user resizes the browser to a small height, your links will look cramped together because then 5% of that small height becomes very small.

It's better to use vh for larger boxes on the page, for example the height of a container that wraps the main content of the page and so on.

Reply:
Thank you, Babak